### ISSUE TITLE
Multiple keyboard enhancements

### ISSUE THREAD

**Issue by paride** [icasdri/Mather#14]
I like the possibility of customizing the keyboard that came with the latest version of Mather, but I think it wastes the usefulness of the long-tap action. I have a few ideas about how to improve the keyboard, keep what you like and discard the rest. I hope it helps.
- [ ] Use the long-tap action to access a (customizable) "2nd" function of the keys. For example, if I assign one key to the sin() function, I could assign arcsin() to the long-tap;
- [ ] In order to customize the keyboard add a "keyboard customization mode" in the settings menu;
- [ ] Let keys be labelled in a way that is different from their actual function. For example I'd like to have a key labelled "sin" that actually types "sin(", or a key labellel "√" that types "sqrt(";
- [x] Do not uppercase everything in the keyboards. I want to see "sin", not "SIN";
- Allow the customization of the number of columns and rows;
- Optionally show the key outline (it's easier to type!)

You probably already had some of these idea, it's nothing new after all.
Thank you for this app, finally a good, maintained, FLOSS scientific calculator!

**Comment by paride** [comment]
Is this project still alive?

**Comment by icasdri** [comment]
Yes, I just haven't had much time to work on it. Contributions are always welcome, however.
